src/Rector/v9/v3/ValidateAnnotationRector.php

```diff
@@ -128,24 +153,20 @@ private function createPropertyAnnotation(string $validatorAnnotation): PhpDocTa
             }
 
             $annotation = sprintf(
-                '@TYPO3\CMS\Extbase\Annotation\Validate("%s", options={%s})',
+                '@Extbase\Validate("%s", options={%s})',
                 trim($validator),
                 implode(', ', $optionsArray)
             );
         } else {
-            $annotation = sprintf('@TYPO3\CMS\Extbase\Annotation\Validate(validator="%s")', $validatorAnnotation);
+            $annotation = sprintf('@Extbase\Validate("%s")', $validatorAnnotation);
         }
         return new PhpDocTagNode($annotation, $this->createEmptyTagValueNode());
     }
 
     private function createMethodAnnotation(string $validatorAnnotation): PhpDocTagNode
     {
         [$param, $validator] = explode(' ', $validatorAnnotation);
-        $annotation = sprintf(
-            '@TYPO3\CMS\Extbase\Annotation\Validate(validator="%s", param="%s")',
-            $validator,
-            ltrim($param, '$')
-        );
+        $annotation = sprintf('@Extbase\Validate(validator="%s", param="%s")', $validator, ltrim($param, '$'));
```

Comment:
This line seams to be not covered by tests, at least I don't see any `validator=` in the tests.